tracee/tracee.go

```diff
@@ -205,6 +205,14 @@ func New(cfg TraceeConfig) (*Tracee, error) {
 			t.mntNsFirstPid[uint32(mnt_ns)] = 1
 		}
 	}
+
+	if err := os.MkdirAll(t.config.OutputPath, 0755); err != nil {
```

Comment:
Can we now remove mkdir from capture exec and capture write?

Reply:
no, because it's creating the mount subdirectory there